test/helpers/manage-merge-queue.test.ts

```diff
@@ -69,6 +73,33 @@ describe('manageMergeQueue', () => {
     });
   });
 
+  describe('pr not core approved case', () => {
+    const queuedPrs = [{ labels: [{ name: READY_FOR_MERGE_PR_LABEL }] }, { labels: [{ name: READY_FOR_MERGE_PR_LABEL }] }];
+    beforeEach(async () => {
+      (octokit.pulls.list as unknown as Mocktokit).mockImplementation(async ({ page }) => ({
+        data: page === 1 ? queuedPrs : []
+      }));
+      (octokit.pulls.get as unknown as Mocktokit).mockImplementation(async () => ({
+        data: {
+          merged: false,
+          head: { sha: 'sha' },
+          number: 123,
+          labels: [{ name: READY_FOR_MERGE_PR_LABEL }, { name: 'QUEUED FOR MERGE #1' }]
+        }
+      }));
+      (approvalsSatisfied as jest.Mock).mockResolvedValue(false);
+      await manageMergeQueue();
+    });
+
+    it('should check for approvals satisfied', () => {
+      expect(approvalsSatisfied).toHaveBeenCalledWith();
+    });
```

Comment:
Instead of this assertion, let's assert that `manageMergeQueue()` is not called

Reply:
But that's the function we are calling, right? Did you mean `setCommitStatus` not being called instead?

Reply:
Oh whoops 🤦 yes sorry!